lib/pages/search/search.dart

```diff
@@ -58,6 +58,9 @@ class SearchController extends State<Search> {
 
   String get searchWord => textEditingController.text;
 
+  bool get isSearchWordUserId =>
```

Comment:
Can you find a better name for this? Still not understand what is it? What is use for?